### DISCUSSION TITLE
Direction on how to setup multi user access for self hosted?

### DISCUSSION THREAD

**Discussion by patiljignesh** [Drakkar-Software/OctoBot#2516] [Q&A] [answered]
Hi there,

I am aware of the fact that an assumption made is that self-hosted OctoBot is for single user only (https://www.octobot.cloud/en/guides/octobot-interfaces/web). 

How do I go about creating a user management of multiple users, and does anyone have any ideas or hints on how to get started? 

Thank you so much in advance. 

Regards

**Comment by jimoquinn** [comment, marked as answer]
I use the binary version and when you start it up, it'll automatically create a set of files.  One of those files will be user/config.json and in that file you can set the port number so you can run multiple Octobots on the same system.  

I'm sure there is a better way, but here is what I did.  I created two directories, copied the binary into each of them, and started it up for a few seconds, then stopped it.  Then edited user/config.json and found the part like this: 

```  
"services": {
        "web": {
            "auto-open-in-web-browser": true,
            "password": "",
            "port": 5001,
            "require-password": false
        }
    }
```
I may have added the "port": 5001, but regardless, you'll need that in the file.  In one directory you can use port 5001 and in the other directory set it to 5002.  Then start up each Octobot and access them like http://192.168.1.2:5001 and http://192.168.1.2:5002.  Change your IPs accordingly.

BTW, I didn't see a way to add multiple accounts to a single instance of Octobot.
